### PR TITLE
[api] Implement TextFormatter.

### PR DESCRIPTION
**Pull request by joojis**

### CODE REVIEW

File: beaver_api/lib/src/formatter/text_formatter.dart [hunk 1]
```diff
@@ -10,5 +11,30 @@ class TextFormatter implements Formatter {
 
   TextFormatter(this._result);
 
-  String toText() => _result.toString();
+  String toText() {
+    var buffer = new StringBuffer();
```
Comment (maniagoon) [inline beaver_api/lib/src/formatter/text_formatter.dart]: Use final instead of buffer.

File: beaver_api/lib/src/formatter/text_formatter.dart [hunk 2]
```diff
@@ -10,5 +11,30 @@ class TextFormatter implements Formatter {
...
+
+    buffer.writeln("${_result.project.name} (${_result.project.id})");
+
+    var items = {
```
Comment (maniagoon) [inline beaver_api/lib/src/formatter/text_formatter.dart]: Use final instead of var.

File: beaver_api/lib/src/formatter/text_formatter.dart [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import 'package:beaver_trigger_handler/beaver_trigger_handler.dart';
 
 import '../formatter.dart';
+import 'package:beaver_task/src/run.dart';
```
Comment (maniagoon) [inline beaver_api/lib/src/formatter/text_formatter.dart]: Optimize Imports.

**Comment by joojis** [comment]
default

**Pull request merged**